### DISCUSSION TITLE
How to fix position from bottom

### DISCUSSION THREAD

**Discussion by vesal** [nextapps-de/winbox#134] [Ideas]
If I use mobile and drag window to bottom of page and turn mobile to landscape the
window does not show.  Sometimes it would be idea to measure the position from
bottom of screen instead top.

If I create the window with y: "bottom" and turn the mobile landscape, the window does not show either.
Would it be idea to measure y-coordinate from bottom of screen if started by y: "bottom"?

**Comment by ts-thomas** [comment]
This will ends up in the same situation when the window was dragged to the upper corner. What we need is a resize listener + orientationchange listener to bring back window into the available viewport.

**Comment by vesal** [comment]
Somehow we need to bring the window back to bottom when
returned from landscape to portrait.  So just moving the window
inside viewport is not enough.  Maybe areas in viewport where
dragged window is fixed to bottom or left side, right side or top or
even center. When dragged elsewhere, it's position is numeric fixed
and then moved inside viewport.  Or remembered different places
in landscape and portrait.

**Comment by ts-thomas** [comment]
This should be fixed now.
